.github/workflows/windows.yml

```diff
@@ -199,3 +199,109 @@ jobs:
           name: oqs-provider-msvc
           path: D:/a/oqs-provider/oqs-provider/_build/lib/oqsprovider.dll
 
+  msvc_native:
+# Run a job for each of the specified target architectures:
+    strategy:
+      matrix:
+        os:
+          - windows-2019
+          - windows-2022
+        platform:
+          - arch: win64
+            oqsconfig: -DOQS_ALGS_ENABLED=STD
+            osslconfig: no-shared no-fips VC-WIN64A
+        toolchain: 
+#        - .CMake/toolchain_windows_x86.cmake
+        - .CMake/toolchain_windows_amd64.cmake
+        msarch:
+          - x64
+# - arm64
```

Comment:
Am I assuming correct that github CI doesn't support these architecture types? Worthwhile adding a comment? Or completely eliminate these 2 lines?